src/components/Comments/ReviewComments.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { getCommentsOnReview } from '../../utils/api'
+
+const ReviewComments = () => {
+  const [reviewComments, setReviewComments] = useState([])
+  const [isLoading, setIsLoading] = useState(true)
+  const { review_id } = useParams()
+
+  useEffect(() => {
+    setIsLoading(true)
+    getCommentsOnReview(review_id).then((reviewCommentData) => {
+      setReviewComments(reviewCommentData)
+      setIsLoading(false)
+    })
+  }, [review_id])
+
+  return isLoading ? (
+    <h2>ReviewComments: Loading Animation Goes Here</h2>
```

Comment:
Don't focus too much on this animation plan! Just a h2 or p tag is fine. Leave it for the end so you don't get distracted